### PR TITLE
Artifact Meteors, Monolith Fixes

### PR DESCRIPTION
**Pull request by CollenN**
This fixes a bunch of things that were not working correctly with monoliths, so now they should spawn more consistently. Miners now also spawn with a paper that explains how monoliths work. GPSes now also track activated monoliths, so they're more useful in the mines.

This also adds artifact meteors, which will occasionally hit the station unannounced, similar to space dust. They leave behind artifacts once they hit the station!

Also, the "travel" artifact power has been tweaked a bit to be less frustrating and random. Now you'll always teleport back to the station if you're not on the station z-level.

Lizard fixes... aren't in this commit so I'm not sure why I named the branch that. SHRUG

### CODE REVIEW

**Comment by CollenN** [comment]
Still need to fix merge conflicts on this as soon as I remember how to do that

**Comment by CollenN** [comment]
Ok fixed the merge conflicts hopefully

**Comment by Profakos** [comment]
Got to examine this thoroughly

File: code/game/asteroid.dm [hunk 1]
```diff
@@ -173,7 +174,7 @@ var/global/list/spawned_surprises = list()
 			y_size = 5
 			areapoints = 25
 
-	if(!monolith)
+	if(monolith != 1)
```
Comment (Profakos) [inline code/game/asteroid.dm]: !monolith is the same as monolith == 0 and monolith == null in scripting languages, which is neater

File: code/game/objects/items/devices/instruments.dm [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
-	desc = "It's made of wood and has bronze strings."
-	icon = 'icons/obj/musician.dmi'
-	icon_state = "guitar"
-	item_state = "guitar"
-	force = 10
-	attack_verb = list("played metal on", "serenaded", "crashed", "smashed")
-	hitsound = "swing_hit"
-	instrumentId = "guitar"
-
+//copy pasta of the space piano, don't hurt me -Pete
```
Comment (Profakos) [inline code/game/objects/items/devices/instruments.dm:1]: Please check if these two files contents are the same or not. If they are same, they conflicted with the line endings, so nothing non-whitespace changed, so this change can be discarded. Same with the other files

File: code/modules/events/sayu_artifactdust.dm [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// ARTIFACT METEORS -- Collen
+
+/datum/round_event_control/meteor_wave/artifact
+	name = "Artifact Meteor"
+	typepath = /datum/round_event/meteor_wave/artifact
+	weight = 260 // This value needs some tweaking, not really sure what to put it at.
```
Comment (Profakos) [inline code/modules/events/sayu_artifactdust.dm]: this makes artifacts as common as _space dust_, happening every five minutes.
An actual event is usually around 5-10

File: code/modules/mining/artifact/powers.dm [hunk 4]
```diff
@@ -925,10 +925,11 @@
 			user.visible_message("<span class='danger'>[user] fires [src]!</span>", "<span class='danger'>You fire [src]!</span>", "You hear a burst of energy!")*/
 		//if(A_MAGICPROJ) -- TODO
 		if(A_TRAVEL)
-			var/list/pos_levels = list(1,1,1,3,4,5,6) // Travels to the station the majority of the time.
-			for(var/num in pos_levels) // Don't travel to the same z-level
-				if(target.z == num)
-					pos_levels -= num
+			var/list/pos_levels = list()
+			if(user.z == 1)
+				pos_levels = list(3,4,5,6,7)
```
Comment (Profakos) [inline code/modules/mining/artifact/powers.dm]: please use the Zlevel defines for these. See: https://github.com/tgstation/-tg-station/blob/cf6eca2d472b6e8c2ca80b7ba9f18678a8068173/code/__DEFINES/misc.dm#L115

File: code/game/gamemodes/meteor/meteors.dm [hunk 5]
```diff
@@ -294,6 +296,23 @@
 	if(prob(20))
 		explosion(src.loc,2,4,6,8)
 
+//Artifact Meteor
+/obj/effect/meteor/artifact
+	name = "ancient meteor"
+	icon_state = "artifact"
```
Comment (Profakos) [inline code/game/gamemodes/meteor/meteors.dm:303]: please put it into a new sayu_meteors.dmi ;-;

File: code/modules/events/sayu_artifactdust.dm [hunk 6]
```diff
@@ -0,0 +1,23 @@
+// ARTIFACT METEORS -- Collen
+
+/datum/round_event_control/meteor_wave/artifact
+	name = "Artifact Meteor"
+	typepath = /datum/round_event/meteor_wave/artifact
+	weight = 50 // This value needs some tweaking, not really sure what to put it at.
```
Comment (Profakos) [inline code/modules/events/sayu_artifactdust.dm]: That is still too much for an event like this. The weight of "some lights overload" is 40. Put it between 5 to 10.

**Comment by CollenN** [comment]
It's meant to happen at least somewhat commonly. Like, at least once per round.

**Comment by Profakos** [comment]
You can't ensure At Least Once Per Round though. Well okay, this is common enough then.

**Pull request merged**